### PR TITLE
chore(deps): Bump chromium-bidi to 0.5.12

### PR DESCRIPTION
**Pull request by sadym-chromium**
### Features

* implement `storage.deleteCookies` ([#1915](https://github.com/GoogleChromeLabs/chromium-bidi/issues/1915)) ([18d3d4f](https://github.com/GoogleChromeLabs/chromium-bidi/commit/18d3d4f197320dcb152d9d0b729bcf2a77720cec))

### Bug Fixes

* implement less flaky network module ([#1871](https://github.com/GoogleChromeLabs/chromium-bidi/issues/1871)) ([4ec8bad](https://github.com/GoogleChromeLabs/chromium-bidi/commit/4ec8bad7d1f42b916bbab207ab68371e79063ff1))
* parse `browser.RemoveUserContextParameters` ([#1905](https://github.com/GoogleChromeLabs/chromium-bidi/issues/1905)) ([a50821b](https://github.com/GoogleChromeLabs/chromium-bidi/commit/a50821b2661777be4509bf9bc9c6a0890e5e5a7a))

### CODE REVIEW

**Comment by sadym-chromium** [comment]
Not sure if this description is OK, or I have to wait for dependabot to pick this dependency up.

File: packages/puppeteer-core/package.json [hunk 1]
```diff
@@ -120,7 +120,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@puppeteer/browsers": "2.1.0",
-    "chromium-bidi": "0.5.11",
+    "chromium-bidi": "^0.5.12",
```
Comment (Lightning00Blade) [inline packages/puppeteer-core/package.json]: We need this to be a pinned version.

Comment (sadym-chromium) [reply packages/puppeteer-core/package.json]: right, my bad

**Review by Lightning00Blade — APPROVED** [review]
LGTM, description is OK no need to worry.

File: test/TestExpectations.json [hunk 2]
```diff
@@ -2504,12 +2504,6 @@
     "expectations": ["FAIL"],
     "comment": "Needs investigation, it looks like a bug in Puppeteer"
   },
-  {
-    "testIdPattern": "[network.spec] network Response.fromCache should work",
```
Comment (Lightning00Blade) [inline test/TestExpectations.json:2508]: Please keep an eye. I think it may start to be flaky now.

Comment (sadym-chromium) [reply test/TestExpectations.json:2508]: ack. Marked as flaky for now.

**Pull request merged**